src/shared/components/plus-button/PlusButton.tsx

```diff
@@ -0,0 +1,32 @@
+/**
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { type HTMLAttributes } from 'react';
+import { ReactComponent as PlusIcon } from './plus-icon.svg';
+
+import styles from './PlusButton.scss';
+
+type Props = Omit<HTMLAttributes<HTMLButtonElement>, 'children'>;
+
+const DeleteButton = (props: Props) => {
```

Comment:
I am a bit confused about the naming of this button as `DeleteButton`. Isn't this supposed to be the `PlusButton`?

Reply:
Ouch! 🤦 My bad! Must have copy-pasted code from the DeleteButton component that I had created earlier. I was sure I renamed it!

Reply:
✅